projects/core/components/root/root.style.less

```diff
@@ -1,5 +1,5 @@
-@import '../../styles/mixins/mixins.less';
-@import '../../styles/mixins/text.less';
+@import '@taiga-ui/core/styles/mixins/mixins';
+@import '@taiga-ui/core/styles/mixins/text';
```

Comment:
Why not entire local file?

Reply:
what do you mean?

Reply:
I mean single import of `@taiga-ui/core/styles/taiga-ui-local';`

Reply:
aaa